authorization/invitation/service/invitation_service.go

```diff
@@ -421,16 +440,45 @@ func (s *invitationServiceImpl) Accept(ctx context.Context, token uuid.UUID) (st
 
 			err = s.Repositories().IdentityRoleRepository().Create(ctx, ir)
 			if err != nil {
-				return resourceID, err
+				return resourceID, redirectPath, err
 			}
 		}
 
+		redirectPath, e := s.getRelativeSpacePath(ctx, inviteToResource.ResourceID)
+		if e != nil {
+			return resourceID, redirectPath, err
```

Comment:
@dipak-pawar , for future compatibility with new resource types - 
1. check if resource type is of type space
2. if not - then redirect the user to https://openshift.io/_redirects/_acceptInvitation?status=success
3. if yes - use the space home path 

It should not be flagged as an error here as long the invitation was a valid one.